spec/features/customer_orders/new_spec.rb

```diff
@@ -0,0 +1,54 @@
+require 'rails_helper'
+
+RSpec.describe 'Customer Orders Index' do
+  it 'shows a link to create a new order' do
+    customer = Customer.create!({
+      first_name: 'Chaz',
+      last_name: 'Simons',
+      address: '123 Cool Guy Street',
+      over_21: true,
+      rewards: 1000
+      })
+    visit "/customers/#{customer.id}/orders"
+    click_link 'Create Order'
+  end
+
+  it 'provides a form for entering a new order' do
+    customer = Customer.create!({
+      first_name: 'Chaz',
+      last_name: 'Simons',
+      address: '123 Cool Guy Street',
+      over_21: true,
+      rewards: 1000
+      })
+    visit "/customers/#{customer.id}/orders"
+    click_link 'Create Order'
+
+    fill_in 'special_instructions', with: 'extra onions'
+    fill_in 'number_of_items', with: '5'
+    select "Yes", :from => "paid"
+
+    click_button "Create Order"
```

Comment:
same thing here - make sure you're verifying that you're on the right page and the new info actually exists there after clicking the button